_pages/speaking-resources.md

```diff
@@ -26,21 +28,42 @@ Presenters, regardless of experience, sometimes want a little help. If you’d l
+* These slides will not be shared outside of the organizing team and the captioners. Send the slides to hello AT djangocon DOT us when you are ready. It's perfectly fine if your slides and notes change between the draft and final version.
+* When describing media in your talk (images, screenshots, graphs, videos, etc.), please give a verbal description of what you're showing so that blind or vision-impaired attendees can understand what you're describing.
+  * Example: instead of "here's a graph showing performance improvements with our new async code," say "here's a line graph showing our average response time that decreased from 450 milliseconds to 150 milliseconds after adding our new async code."
+
+### General
+
+* Use gender-neutral language wherever possible. Don't default to using "he" or "him" everywhere.
+* Be respectful of other speakers, your audience, and the community at large, especially when offering critiques. It's possible to say "this needs improvement" without saying "this is bad."
+* Consider including your social media handle(s) on your opening and closing slides! If you have space, you can even include them on the footer of every slide.
+* Consider publishing your slides after your talk (on the platform of your choice) and sharing the link with the #djangocon hashtag on social media.
+* Include a copy of what you plan to say in each slide's notes (if possible). This will help you while you're in presenter view and also make it easier for people who may be looking at your slides after the conference.
+* Before you connect your laptop to the projector, make sure you have every window except for your slides minimized or closed to avoid accidental information disclosure.
+* Make sure you turn on your computer's do not disturb mode before starting your presentation to avoid interruptions.
+* If your computer needs a dongle to connect to an HDMI cable, please bring it. We'll have most of the common ones, including a USB-C/Thunderbolt connector, but it always helps to have one that you know works.
 
 ## Slide Guidelines
 
-* Use Keynote, PowerPoint, LibreOffice, or Google Slides for your slides.
+### Format
+
 * Minimal slides are best—-avoid walls of text and long lists of bullets.
-* Aim for high contrast slides, avoiding colors that may be difficult to see for those with colorblindness. (You can [check your contrast online](http://webaim.org/resources/contrastchecker/); you just need the hex codes for your colors!)
 * Light background with dark text is easiest to read; be mindful that the projection screen is white.
-* Make text as large as possible, at least 68pt.
-* Choose fonts with adequate spacing between letters, and avoid thin or cursive fonts.
-* Leave the bottom ⅓ of your slides free of text to ensure nothing is obscured.
+ Choose fonts with adequate spacing between letters, and avoid thin or cursive fonts.
+* Leave the bottom 10⅓ of your slides free of text to ensure nothing is obscured by other attendees' heads.
```

Comment:
> 10⅓

Did you mean ⅓?

Reply:
Yep, that was me failing at reading VS Code's font. I thought that was a percent sign.

Reply:
I changed it to 25% to make it a little less onerous.